app/services/person_with_remote_fields.rb

```diff
@@ -0,0 +1,49 @@
+class PersonWithRemoteFields < SimpleDelegator
+  REMOTE_FIELDS = [
```

Comment:
Eventually, I'd like `permitted_person_params` to come from this class, rather than `Person`. I think `Person` shouldn't know anything about remote fields.